charts/reform-scan-blob-router/values.yaml

```diff
@@ -15,7 +15,11 @@ java:
         - storage-account-secondary-key
         - bulkscan-storage-account-name
         - bulkscan-storage-account-primary-key
+        - flyway-password
   environment:
+    FLYWAY_URL: jdbc:postgresql://blob_router-{{ .Values.global.environment }}.postgres.database.azure.com:5432/bulk_scan?sslmode=require
+    FLYWAY_USER: blob_router@blob_router-{{ .Values.global.environment }}
+    FLYWAY_SKIP_MIGRATIONS: true
```

Comment:
thought it would be clearer than `FLYWAY_NOOP_STRATEGY`

Reply:
this value should it be false for preview ?

Reply:
updated preview template file